File: MK/10월 3주차/H-index.java
import java.util.*;

class Solution {
    public int solution(int[] citations) {

        int res = citations.length;

        Arrays.sort(citations);

        for (int i=0; i<citations.length; i++) {
            if (res <= citations[i]) {
                return res;
            }
            res--;
        }

        return res;
    }
}

File: DB/10월 3주차/H-Index.js
function solution(citations) {
  //가장 많이 된거 부터 정렬 (내림차순)
  citations.sort((a, b) => b - a);
  let h = 0;
  for (let i = 0; i < citations.length; i++) {
    if (citations[i] > i + 1) {
      h = i + 1;
    } else {
      break;
    }
  }
  return h;
}
